Runtime/RenderPipeline/PSXRenderPipeline.cs

```diff
@@ -879,6 +880,30 @@ static void PushFogParameters(Camera camera, CommandBuffer cmd)
                 }
             }
         }
+        
+        static void PushAnalogSignalParameters(Camera camera, CommandBuffer cmd, PSXRenderPipelineAsset asset)
+        {
+            using (new ProfilingScope(cmd, PSXProfilingSamplers.s_PushPrecisionParameters))
```

Comment:
Wrong profiling scope. Don't use s_PushPrecisionParameters here. You need to define your own. Otherwise profilers and debuggers will tag this code as the wrong thing.